internal/config/config.go

```diff
@@ -105,9 +127,11 @@ type ConfigPolicyExclusions struct {
 }
 
 type K8SApiConfig struct {
-	Enabled     bool     `yaml:"enabled"`
-	ApiVersions []string `yaml:"resourceAPIVersions"`
-	MaxQPS      int      `yaml:"clientMaxQPS"`
+	Enabled        bool     `yaml:"enabled"`
+	ApiVersions    []string `yaml:"resourceAPIVersions"`
+	MaxQPS         int      `yaml:"clientMaxQPS"`
+	TimeoutSeconds int      `yaml:"clientTimeoutSeconds"`
+	MaxGoroutines  int      `yaml:"maxGoroutines"`
```

Comment:
I doubt it will be ever used by anyone else than us, to be honest :-)
I've added it here for completeness and to allow us to tweak the setting easily. However I agree that this is not consistent with the other setting like this. I'm OK to keep only hardcoded defaults.

For common value for all APIs - I don't think this is the best idea as each API can behave differently. So I would leave different defaults just in case.

Reply:
good point, let's keep them different, in defaults

Reply:
I've removed `MaxGoroutines ` from the Config then.
Defaults are in variable in the kubernetes client.